### PR TITLE
Reload redis scripts on reconnect

### PR DESCRIPTION
**Pull request by lpfeup**
fixes https://github.com/gresrun/jesque/issues/100

### CODE REVIEW

File: src/main/java/net/greghaines/jesque/worker/WorkerImpl.java [hunk 1]
```diff
@@ -539,6 +536,11 @@ protected void recoverFromException(final String curQueue, final Exception ex) {
             } else {
                 authenticateAndSelectDB();
                 LOG.info("Reconnected to Redis");
+                try {
+                    loadRedisScripts();
+                } catch (IOException e) {
+                    LOG.error("Failed to load redis scripts", e);
```
Comment (ofirnk) [inline src/main/java/net/greghaines/jesque/worker/WorkerImpl.java]: Maybe `Failed to reload Lua scripts after reconnect`

Comment (lpfeup) [reply src/main/java/net/greghaines/jesque/worker/WorkerImpl.java]: done

**Comment by lpfeup** [comment]
To test this, I basically initialized and started a `WorkerImpl` and then restarted the redis instance (redis implicitly flushes the script cache on restart).

Without this fix, `WorkerImpl` would terminate with a `JedisNoScriptException` while polling the queues after a redis restart.
With this fix, the worker keeps running after a redis restart.

**Comment by ofirnk** [comment]
Cool, sounds good 👏

**Pull request merged**